docs/security/docs/03-04-roles-in-kyma.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Roles in Kyma
+type: Details
+---
+
+Kyma uses roles and groups to manage access in the cluster. Every cluster comes with 5 predefined roles which give the assigned users different level of permissions suitable for different purposes.
+These roles are defined as ClusterRoles and use the Kubernetes mechanism of aggregation, which allows to combine multiple ClusterRoles into a single ClusterRole. Using the aggregation mechanism allows to efficiently manage access to Kubernetes resources and Kyma-specific resources.
```

Comment:
```suggestion
These roles are defined as ClusterRoles and use the Kubernetes mechanism of aggregation which combines multiple ClusterRoles into a single ClusterRole. Use the aggregation mechanism to efficiently manage access to Kubernetes and Kyma-specific resources.
```

Reply:
Hope the meaning is preserved? If not just pls change to `allows you to` ...

Reply:
I don't think the mechanism combines - the person who defines the roles combines them using the mechanism of aggregation. I'd stick with the original.

Reply:
In that case `you` is missing here